@ecomplus/storefront-components/src/js/SearchEngine.js

```diff
@@ -405,6 +405,11 @@ export default {
         }
       }
       this.fetchItems()
+    },
+    setSortOrder (sort) {
```

Comment:
```suggestion

    setSortOrder (sort) {
```

mantenha o code style por favor